spec/api/channels_spec.cr

```diff
@@ -60,6 +60,30 @@ describe LavinMQ::HTTP::ChannelsController do
         end
       end
     end
+
+    it "should return get_no_ack count 1" do
+      with_http_server do |http, s|
+        with_channel(s) do |c|
+          vh_name = "/"
+          q = c.queue
+          q.publish "get_no_ack_message"
+          message = q.get(true)
+          if message = nil
+            fail "No message returned"
+          end
```

Comment:
This isn't needed - if no message is returned the assertions below would fail because `get_no_ack` would be 0

Reply:
so you can also skip assigning message, just do `q.get(true)`

Reply:
Hmm yeah maybe that was redundant. It was mainly to deal with `nil` scenarios - same [as this](https://github.com/cloudamqp/lavinmq/pull/1099#discussion_r2097265991)